sdk/servicebus/azure-messaging-servicebus/src/test/java/com/azure/messaging/servicebus/ServiceBusClientBuilderTest.java

```diff
@@ -243,6 +256,103 @@ public void testConnectionStringWithSas() {
                 .connectionString("Endpoint=sb://sb-name.servicebus.windows.net/;EntityPath=sb-name"));
     }
 
+    @Test
+    public void sendAndReceiveEventByAzureNameKeyCredential() {
+        ConnectionStringProperties properties = getConnectionStringProperties();
+        String fullyQualifiedNamespace = getFullyQualifiedDomainName();
+        String sharedAccessKeyName = properties.getSharedAccessKeyName();
+        String sharedAccessKey = properties.getSharedAccessKey();
+        String queueName = getQueueName(0);
+
+        final ServiceBusMessage testData = new ServiceBusMessage(TEST_MESSAGE.getBytes(UTF_8));
+
+        ServiceBusSenderAsyncClient senderAsyncClient = new ServiceBusClientBuilder()
+            .credential(fullyQualifiedNamespace, new AzureNamedKeyCredential(sharedAccessKeyName, sharedAccessKey))
+            .sender()
+            .queueName(queueName)
+            .buildAsyncClient();
+        try {
+            StepVerifier.create(
+                senderAsyncClient.createMessageBatch().flatMap(batch -> {
+                    assertTrue(batch.tryAddMessage(testData));
+                    return senderAsyncClient.sendMessages(batch);
+                })
+            ).verifyComplete();
+        } finally {
+            senderAsyncClient.close();
+        }
+    }
+
+
+    @Test
+    public void sendAndReceiveEventByAzureSasCredential() {
```

Comment:
It says sendAndReceiveEvents but the test is only sending  and receiving.

Reply:
According to your Comment, change method name to `testBatchSendEventByAzureSasCredential`